hack/verify-spelling.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########
+# This script verifies mispellings in location. Today it only supports
+# verifying English locale but can be modified in a future to support
+# other locales also
+# You need to run this script inside the root directory of "website" git repo.
+#
+# Syntax: verify-spelling.sh LOCALE
+# Example: verify-spelling.sh en
+# If no locale is passed, it will assume "en"
+#
+# Requirements:
+# - go v1.14 or superior version
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TOOL_VERSION="v0.3.4"
+
+# cd to the root path
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+
+# create a temporary directory
+TMP_DIR=$(mktemp -d)
+
+# cleanup
+exitHandler() (
+  echo "Cleaning up..."
+  rm -rf "${TMP_DIR}"
+)
+trap exitHandler EXIT
+
+function check() {
+  #NOTE we use go install refer the go site :https://go.dev/doc/go-get-install-deprecation
+  go install github.com/client9/misspell/cmd/misspell@${TOOL_VERSION}
+  cd "${ROOT}"
+  # check spelling
+  echo "Checking spelling..."
+  ERROR_LOG="${TMP_DIR}/errors.log"
+
+  find . -type f | grep -v .golangci.yaml | grep -v vendor/ | xargs misspell >"${ERROR_LOG}"
```

Comment:
We should have a list of allowed words, not the whole document excluded.

Reply:
thanks for review.

you means like 
```
skipping_file="${KUBE_ROOT}/scripts/.spelling_failures"
failing_packages=$(sed "s| | -e |g" "${skipping_file}")
```, 
And we add the ```.golangci.yaml``` and ```vendor/``` to it?

Actually before i did it like that,but error shows, return i update the code ..😂😂

Reply:
For more tools details see https://github.com/client9/misspell
Refer #514 to add `verify-spelling.sh` and `update-spelling.sh`

Reply:
thanks .
I will do it later.